jctools-core/src/main/java/org/jctools/queues/atomic/MpmcAtomicArrayQueue.java

```diff
@@ -92,31 +156,35 @@ public boolean offer(final E e) {
             throw new NullPointerException();
         }
         final int mask = this.mask;
-        final int capacity = mask + 1;
```

Comment:
This is an arbitary difference that retyping it to a long makes consistent with the original unsafe queues. Has no impact.